test/unit/cli/test_std_options.py

```diff
@@ -102,6 +102,29 @@ def test_select_std_options_with_override():
     assert not opts[StdParams.alter_system.name].default
 
 
+def test_select_std_options_with_formatter():
+    container_url_arg = 'container_url'
+    container_name_arg = 'container_name'
+    url_format = "https://my_service_url/{version}/page"
+    name_format = "my_service_name"
+    version = '4.5.6'
+    expected_url = url_format.format(version=version)
+    expected_name = name_format
+
+    def func(**kwargs):
+        assert kwargs[container_name_arg] == expected_name
+        assert kwargs[container_url_arg] == expected_url
```

Comment:
Unused?
```suggestion
```

Reply:
It is used in line 123. It's a callback function that checks the passed parameters.

Reply:
sry, my bad overlooked it.